# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: netwerk/cookie/CookieParser.h
# Commit: fe6e63a2f914
# Full Hash: fe6e63a2f9141d9eeec41bfde52cc34b33c170fe
# Author: Andrea Marchesini <amarchesini@mozilla.com>
# Date: 2025-10-09 14:54 +0000
# Description:
#   Bug 1992536 - Use cookie creation time instead or PR_Now() to validate the expiration time, r=valentin,cookie-reviewers
# ==============================================================================

diff -r 0147060f88c2 -r fe6e63a2f914 netwerk/cookie/CookieParser.h
--- a/netwerk/cookie/CookieParser.h	Thu Oct 09 14:36:53 2025 +0000
+++ b/netwerk/cookie/CookieParser.h	Thu Oct 09 14:54:32 2025 +0000
@@ -70,10 +70,10 @@
   void ParseAttributes(nsCString& aCookieHeader, nsACString& aExpires,
                        nsACString& aMaxage, bool& aAcceptedByParser);
 
-  // aCurrentTime is in milliseconds, and expiry will be stored in milliseconds.
+  // expiry will be stored in milliseconds.
   bool GetExpiry(CookieStruct& aCookieData, const nsACString& aExpires,
-                 const nsACString& aMaxage, int64_t aCurrentTime,
-                 const nsACString& aDateHeader, bool aFromHttp);
+                 const nsACString& aMaxage, const nsACString& aDateHeader,
+                 bool aFromHttp);
 
   static bool CheckAttributeSize(const nsACString& currentValue,
                                  const char* aAttribute,